moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -262,20 +262,19 @@ def get_attached_objects(self, object_ids = []):
             aobjs[key] = msg
         return aobjs
 
-    def check_object_does_not_exist(self, object_name, timeout=4):
+    def wait_for_object_removed(self, object_name, timeout=4):
```

Comment:
Are all of these convenience functions actually useful or should the user directly call `wait_for_state` method.